TLM/TLM/UI/SubTools/LaneArrowTool.cs

```diff
@@ -340,6 +365,22 @@ public LaneArrowTool(TrafficManagerTool mainTool)
             }
 
             GUILayout.EndHorizontal();
+
+            if (CanReset(SelectedSegmentId, (bool)startNode)) {
+                if (GUILayout.Button(
+                    "     Reset     ", // intentionally big to avoid confusion
```

Comment:
Localisation?
Spaces should not be included into the localisation strings. Use larger width if you can.

Reply:
I auto center localized string
![Cities_sKV2A5Z8xL](https://user-images.githubusercontent.com/26344691/73946600-d7f0d880-48fe-11ea-9872-e7c4a6f71f4f.jpg)